climada/hazard/test/test_base.py

```diff
@@ -1104,6 +1105,35 @@ def test_vector_to_raster_pass(self):
         self.assertTrue(haz_fl.fraction.min() >= 0)
         self.assertTrue(haz_fl.fraction.max() <= 0.5 / 2)
 
+
+class TestClear(unittest.TestCase):
+    """Test clear method"""
+
+    def test_clear(self):
+        """Clear method clears everything"""
+        haz1 = Hazard('TC')
+        haz1.read_excel(HAZ_TEMPLATE_XLS)
+        haz1.clear()
+        haz1.centroids.size
+        self.assertEqual([val for key, val in haz1.tag.__dict__.items()], ['', '', ''])
+        self.assertEqual(haz1.units, '')
+        self.assertEqual(haz1.centroids.size, 0)
+        self.assertEqual(len(haz1.event_name), 0)
+        for attr in list(set(haz1.__dict__.keys()).difference({'tag', 'units', 'event_name', 'pool'})):
+            self.assertEqual(haz1.__dict__[attr].size, 0)
```

Comment:
Avoid using the double-underscore notation as much as possible:
```suggestion
        for attr in vars(haz1).keys():
            if attr not in ['tag', 'units', 'event_name', 'pool']:
                self.assertEqual(getattr(haz1, attr).size, 0)
```